mobly/controllers/android_device_lib/jsonrpc_client_base.py

```diff
@@ -214,7 +217,7 @@ def connect(self, port, addr='localhost', uid=UNKNOWN_UID,
                     logging.exception("Failed to create socket connection!")
                     raise
                 time.sleep(1)
-
+        self.port = port
```

Comment:
maybe we should name the arg more specifically.
Is this the port on the device-side?